operator/pkg/reconciliation/construct_statefulset.go

```diff
@@ -7,20 +7,22 @@ package reconciliation
 
 import (
 	"fmt"
-
 	api "github.com/datastax/cass-operator/operator/pkg/apis/cassandra/v1beta1"
 	"github.com/datastax/cass-operator/operator/pkg/httphelper"
 	"github.com/datastax/cass-operator/operator/pkg/images"
 	"github.com/datastax/cass-operator/operator/pkg/oplabels"
-	"github.com/datastax/cass-operator/operator/pkg/utils"
 	"github.com/datastax/cass-operator/operator/pkg/psp"
+	"github.com/datastax/cass-operator/operator/pkg/utils"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
 
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 )
 
+const ZONE_LABEL = "failure-domain.beta.kubernetes.io/zone"
```

Comment:
the linter will complain if there's all caps or underscores, and we've been following that 🤷

```suggestion
const ZoneLabel = "failure-domain.beta.kubernetes.io/zone"
```

sorry!

Reply:
No worries my bad, switching between Python and Go at this time lol. Will fix it in the next 10 min 😉